src/tplink-safeloader.c

```diff
@@ -1335,6 +1338,7 @@ static struct device_info boards[] = {
 		.vendor = "",
 		.support_list =
 			"SupportList:\r\n"
+			"{product_name:Archer C60,product_ver:2.0.0,special_id:43410000}\r\n"
 			"{product_name:Archer C60,product_ver:2.0.0,special_id:42520000}\r\n"
```

Comment:
You sorted the first three changes, but not the last two.

Reply:
whoops, V2 and V3 now have sorted IDs!